src/pages/account/your-beacon-registry-account.tsx

```diff
@@ -39,12 +39,156 @@ export interface YourBeaconRegistryAccountPageProps {
   beacons: AccountListBeacon[];
 }
 
+export interface LegacyBeaconsNotificationProps {
+  beacons: AccountListBeacon[];
+  accountHolderDetails: AccountHolder;
+}
+
+const LegacyBeaconsNotification: FunctionComponent<LegacyBeaconsNotificationProps> =
```

Comment:
This is a big ol' component!  Even though it's only used on this page, could you extract it to its own file in `.../components/domain` to reduce vertical scroll?